.github/workflows/linux.yml

```diff
@@ -11,7 +11,7 @@ on:
 
 jobs:
   validate:
-    name: Validate ghc-${{ matrix.ghc }}
+    name: Validate linux ghc-${{ matrix.ghc }}
```

Comment:
This is ok now but I was plannig to unify linux and macos so this would be `${{ matrix.os }}`

Reply:
yeah thas was my idea too